### DISCUSSION TITLE
资金取回再投入的计算问题

### DISCUSSION THREAD

**Discussion by yxd123a** [famland/demo#19] [Ideas]
之前做过市的法姆兰，再次做市时，收益为什么不是从0开始计算呢？例如：①基本田资金取回一段期间，再次投入资金做市，按当时的APY收益直接从＄11开始计算？（还有为什么再次做市只能最多投入343+资金了呢）②还有野田也是一样，当时只有基本田做市，加入野田做市一万U后，立马收益＄15变成了＄150？

**Comment by otto-001** [comment]
这个问题已经修复，请在复测！
